docs/getting_started/command_line.rst

```diff
@@ -0,0 +1,344 @@
+
+    Name the new expectation suite [npi.warning]: npi.warning
+    A new Expectation suite 'npi.warning' was added to your project
+    Because you requested an empty suite, we\'ll open a notebook for you now to edit it!
+    If you wish to avoid this you can add the `--no-jupyter` flag.
+
+    [I 14:55:15.992 NotebookApp] Serving notebooks from local directory: /Users/dickens/Desktop/great_expectations/uncommitted
+    ... (jupyter opens)
+
+
+``great_expectations suite new --empty --no-jupyter``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you prefer to disable Great Expectations from automatically opening the generated jupyter notebook, add the ``--no-jupyter`` flag.
+
+.. code-block:: bash
+
+    $ great_expectations suite new --empty --no-jupyter
+
+    Enter the path (relative or absolute) of a data file
+    : data/npi.csv
+
+    Name the new expectation suite [npi.warning]: npi.warning
+    A new Expectation suite 'npi.warning' was added to your project
+    To continue editing this suite, run jupyter notebook /Users/taylor/Desktop/great_expectations/uncommitted/npi.warning.ipynb
+
+You can then run juypter yourself.
+
+
+``great_expectations suite edit``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Edit an existing expectation suite.
+Just as writing SQL queries is far better with access to data, so are authoring expectations.
+These are best authored interactively against some data.
+This best done in a jupyter notebook.
+
+.. note::
+  BatchKwargs define what data to use during editing.
+
+   - When suites are created through the CLI, the original batch_kwargs are stored in a piece of metadata called a citation.
+   - The edit command uses the most recent batch_kwargs as a way to know what data should be used for the interactive editing experience.
+   - It is often desirable to edit the suite on a different chunk of data.
+   - To do this you can edit the batch_kwargs in the generated notebook.
+
+To this end, this command interactively helps you choose some data, generates a working jupyter notebook, and opens up that notebook in jupyter.
+
+.. code-block:: bash
+
+    $ great_expectations suite edit npi.warning
+    [I 15:22:18.809 NotebookApp] Serving notebooks from local directory: /Users/dickens/Desktop/great_expectations/uncommitted
+    ... (juypter runs)
+
+.. important::
+
+    Great Expectations generates working jupyter notebooks when you make new suites and edit existing ones.
+    This saves you tons of time by avoiding all the necessary boilerplate.
+
+    Because these notebooks can be generated at any time from the expectation suites (stored as JSON) you should **consider the notebooks to be entirely disposable artifacts**.
+
+    They are put in your ``great_expectations/uncommitted`` directory and you can delete them at any time.
+
+    Because they can expose actual data, we stronly suggest leaving them in the ``uncommitted`` directory to avoid potential data leaks into source control.
```

Comment:
```suggestion
    Because they can expose actual data, we strognly suggest leaving them in the ``uncommitted`` directory to avoid potential data leaks into source control.
```